plugins/telemetry-otel/build.gradle

```diff
@@ -51,7 +54,29 @@ thirdPartyAudit {
     'io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties',
     'io.opentelemetry.sdk.autoconfigure.spi.logs.ConfigurableLogRecordExporterProvider',
     'io.opentelemetry.sdk.autoconfigure.spi.metrics.ConfigurableMetricExporterProvider',
-    'io.opentelemetry.sdk.autoconfigure.spi.traces.ConfigurableSpanExporterProvider'
+    'io.opentelemetry.sdk.autoconfigure.spi.traces.ConfigurableSpanExporterProvider',
+    'org.assertj.core.api.AbstractAssert',
```

Comment:
The `org.assertj.core` are test classes, please remove them (the build will pass once the correct scope is applied to test artifacts).